### DISCUSSION TITLE
What are your goals and expectations of the GitHub Overview bootcamp?

### DISCUSSION THREAD

**Discussion by davidgitit** [Drook-tech/Bootcamp#3] [Q&A]
Are you brand new to GitHub and just want to learn?

Are you an expect with other DevOps offerings such as Azure DevOps, JIRA, Jenkins, GitLab?

Any thing else?

**Comment by dkalmin** [comment]
I have been using Azure DevOps since 2006 when it was release as Team Foundation Server 2005 and I really want to learn all of GitHub features

**Comment by s-samadi** [comment]
I have been using various commercial code scanning tools on the market today and want to see how GHAS compares

**Comment by xavierxmorris** [comment]
I have been using BBS since 2013 when it was release as Stash and I really want to learn all of GitHub features and codespace

**Comment by SachinKab** [comment]
brand new to GitHub and just want to learn

**Comment by prabhkaur1977** [comment]
I am aware of Teams foundation and DevOps. Want to learn the features of Github.

**Reply by Takeers** [reply]
New to Github. Want to learn and share

**Comment by craigmoyle** [comment]
Have used GitHub before and also DevOps - interested in the advanced security features and to complete certification

**Comment by alaurie** [comment]
Learning Python and want to learn Github actions to help automate testing / deployment.

**Comment by theBrecko** [comment]
Rather new to Github.  Been using Azure DevOps for a few years.

**Comment by Gurenato** [comment]
I have been using for a while, but not hands-on. But I have learning give the engagements with partners. I am heavy user of ADO.

**Comment by wicksipedia** [comment]
I have been using various tools like TFS since 2005 and actively using GH for client projects for the last few years. Interested in getting the most out GHAS and Codespaces

**Comment by Saraquin** [comment]
I have worked a little with various repo systems but more ad-hoc as a support to developers etc and now I will be working with it more for Microsoft Cloud projects but from a Security view.

**Comment by s-aqua** [comment]
Have used GitHub before and Azure DevOps, still fairly new. 

Looking to get more in depth with GitHub features.

**Comment by answerau** [comment]
fairly new to GitHub, want to learn more about everything really!

**Comment by william-liebenberg** [comment]
Long time TFS / VSO / Azure DevOps user. Not new to GitHub, but want to learn more about full e2e project management and development with GitHub. Then be able to redeliver that to our customers :)

**Comment by tw3lveparsecs** [comment]
i have been using Azure DevOps for some time and coming up to speed on GitHub and recently been using GitHub Actions. Really want to know more about GHAS and GitHub migrations. Plus any new  GitHub features i didnt know about 😄

**Comment by EvetteThorp** [comment]
New to Git Hub - come from MS LSP background, here to get an understanding of it so can discuss with customers (who are mostly on VS DevOps Server)

**Comment by ThorSchutze** [comment]
Used Azure Devops for a few years as well as github. Learning to use actions and looking to move customers to github.

**Comment by ramesh2051** [comment]
New to GitHub and just want to learn more about everything

**Comment by swilkodev** [comment]
Have used Azure DevOps for many years and recently GitHub. Looking to learn about more in depth features and migrations to help with certification and moving clients to GitHub.
